melodiction/src/pages/Layout/index.tsx

```diff
@@ -123,15 +124,16 @@ function DarkModeToggle({ darkMode, toggleDarkMode }: { darkMode: boolean; toggl
     </IconButton>
   );
 }
+*/
 
 export type LayoutProps = {
   melodiesStorage: MelodiesStorage
 }
 
 export default function Layout({ melodiesStorage }: LayoutProps) {
   const isDarkMode = localStorage.getItem('darkMode') !== 'false';
-  const [darkMode, setDarkMode] = useState(isDarkMode);
-
+  const [darkMode] = useState(isDarkMode);
+  
```

Comment:
Can't you hardcode directly the theme value in the component and remove this stateful variable? It isn't useful now.

Also please remove the extra spaces you added.